getting_started/openshift-overview.adoc

```diff
@@ -44,10 +44,14 @@ Develop and deploy containerized applications with {product-title}. {product-tit
 
 * **xref:../applications/projects/working-with-projects.adoc#working-with-projects[Work with projects]**: Create projects from the {product-title} web console or OpenShift CLI (`oc`) to organize and share the software you develop.
 
-* **xref:../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[Work with applications]**:
+include::snippets/unified-perspective-web-console.adoc[]
```

Comment:
@opayne1 This file isn't used anymore, because the getting started book doesn't exist anymore. The only reason this file was still here was that it might've been needed to be added to the overview page by @snarayan-redhat.

@snarayan-redhat did you still need this file for the overview book, or should i go ahead and remove it now?